templates/example_cell_clustering.ipynb

```diff
@@ -11,13 +11,15 @@
    "cell_type": "markdown",
```

Comment:
<p>Same thing as after pixel clustering, most of these visualizations are redundant with Zak's tool. After clustering, we will use the #pixel cluster x cell cluster heatmap to do the manual reassignments. After that, we use your remapping function. Then we display the weighted channel x cell cluster heatmaps (3.6 and 3.7 below), and then the overlays (3.8).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Yup, same as with pixel clustering they're just here for verification that nothing broke.</p>